src/core/CheckBox/__tests__/CheckBox.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, fireEvent } from '@testing-library/react';
+import React from 'react';
+import { KuberaThemeProvider } from '../../../theme';
+import { CheckBox } from '../CheckBox';
+
+describe('CheckBox', () => {
+  it('Renders', () => {
+    const { getByTestId } = render(
+      <KuberaThemeProvider platform="kubera-chaos">
+        <CheckBox checked={false} disabled={false} />
+      </KuberaThemeProvider>
+    );
+    let checkbox = getByTestId('checkbox') as HTMLInputElement;
+    fireEvent.click(checkbox);
+    expect(checkbox.checked).toBeFalsy();
```

Comment:
how come if the initial value is false....after clicking it becomes false again (toBeFalsy)
Can you explain the test cases in the comments on what is being done...
I might have misunderstood it too...apologies for that...if that's the case...but regardless explanation for test cases is good